### ISSUE TITLE
support multiple redirect urls

### ISSUE THREAD

**Issue by harti2006** [zalando-stups/mint-worker#2]
It should be possible to create OAuth clients with multiple redirect urls.

**Comment by snackycracky** [comment]
+1

**Comment by hjacobs** [comment]
:+1:

**Comment by s-soroosh** [comment]
+1

**Comment by inikolaev** [comment]
Are there any pointers on what parts needs to be changed?
I found a reference to `redirect_url` at [sync_user.clj#L53](https://github.com/zalando-stups/mint-worker/blob/master/src/org/zalando/stups/mint/worker/job/sync_user.clj#L53) which seems to be a simple fix, but it also looks like a change needs to be made to `mint-storage` to actually store multiple URLs.

**Comment by hjacobs** [comment]
@inikolaev yes, that's correct: `mint-storage` needs to be changed in order to store a list of redirect URIs.
